scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1289,13 +1315,21 @@ private void handleTimeout(DTLSFlight flight) {
 			LOGGER.log(Level.FINE, "Re-transmitting flight for [{0}], [{1}] retransmissions left",
 					new Object[]{flight.getPeerAddress(), max - flight.getTries() - 1});
 
-			flight.incrementTries();
-			sendFlight(flight);
+			try {
+				flight.incrementTries();
+				flight.setNewSequenceNumbers();
```

Comment:
I take a deeper look and it seems we always did that before..
Now we do that only on retransmission.
And when I read the spec, I understand we must do that only the first time (as sequence number is for anti-replay)
So I'm a bit confused ... ^^
(This change is not directly linked to this issue ?)